aneris/cli.py

```diff
@@ -81,6 +81,32 @@ def harmonize(inf, history, regions, rc, output_path, output_prefix):
         logger().info('Writing diagnostics to: {}'.format(fname))
         aneris.pd_write(diagnostics, fname)
 
+def harmonize_db(inf, history, regions, rc, output_path, output_prefix):
```

Comment:
I would suggest splitting `harmonize()` into two functions, e.g., `run_harmonization()` and `write_output()` (as you see fit). In general, we want to try to avoid copying/pasting code when we can identify clear operations that can then be reusable in other contexts. Here, you can then just use the `run_harmonization()` in the database workflow.